go/converters.go

```diff
@@ -39,3 +47,134 @@ func XML(any interface{}) string {
 
 	return string(str)
 }
+
+func SQL(dict DictionaryRepresentable, sqlDialect string) string {
+	var buf bytes.Buffer
+	var sqlCmds string
+	dictId := 1
+	entryId := 1
+	etyId := 1
+	usageId := 1
+	groupId := 1
+	defId := 1
+	exId := 1
+
+	// Generate SQL create statements and constraints
+	generateCmd := &ddl.GenerateCmd{
+		Dialect:   sqlDialect,
+		DirFS:     fsys,
+		Filenames: []string{"sql.go"},
+		DryRun:    true,
+		Stdout:    &buf,
+	}
+	err := generateCmd.Run()
+	if err != nil {
+		fmt.Println(err)
+	}
+	sqlCmds += buf.String()
+
+	// Insert dictionary w/ PK of 1
+	d := sq.New[DICTIONARY]("")
+	insertQuery := sq.
+		InsertInto(d).
+		Columns(d.NAME, d.ID).
+		Values(sq.Literal(dict.Name), sq.Literal(dictId))
+	dict_query, _, err := sq.ToSQL(sq.DialectPostgres, insertQuery, nil)
```

Comment:
Why is `sq.DialectPostgres` hardcoded here and not using the `sqlDialect` passed into the function?

Reply:
Oops, this was hardcoded at the beginning because I was testing only PostgreSQL. This should be updated now!